src/ch02-02-entering-protected-mode.md

```
@@ -6,71 +6,71 @@ _"With great power comes great responsibility." - Voltaire / Spider-Man_
 - What the segment includes. i.e data segment, code segment etc.
 - What is the privilege level of the segment. i.e can anyone access it or only the kernel
 - For a data segment, Is the data read only, or may I modify it?
-- For a code segment, Can I execute it, or not yet.
+- For a code segment, can I execute it or not yet.
 
-If you gussed something that is similar to this, you are mostly correct!
+If you guessed something similar to this, you are mostly correct!
 
 Our entry will look like this:
 <figure style="margin: 0; text-align: center">
   <img src="assets/gdt_struct.svg"></img>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)